docs/resources/replication.md

```diff
@@ -60,6 +60,10 @@ The following arguments are supported:
 
 * **action** - (Required)
 
+* **registry_id** - (Required) The registry ID of the Registry Endpoint
+
+* **speed** - (Optional) The Maximum network bandwidth in Kbps for each execution. Default is `0` (unlimited).
```

Comment:
It would be helpful here to add a unit of measurement? is kbps or mbps`

Reply:
It is mentioned that the unit is Kbps. Do you suggest we need even more clarification?